apps/jupyter/run-jupyter.sh

```diff
@@ -22,18 +31,18 @@ EOF
 fi
 ln -snf "${JUPYTER_CONFIG_DOCKER}" $HOME/.jupyter
 
-rm /app/.local/share/jupyter/runtime/jpserver-*-open.html 2>/dev/null
+find "${RUNTIME_DIR}" -type f -name 'jpserver-*-open.html' -delete 2>/dev/null
```

Comment:
nit: If find doesn't report errors on no matching files, remove the 2>/dev/null

Reply:
It reports errors on directories it lacks permission to traverse.